web_external/Featuresets/FeaturesetsView.js

```diff
@@ -24,6 +24,11 @@ isic.views.FeaturesetsView = isic.View.extend({
         }, this);
         this.featuresets.fetch();
 
+        // TODO: Use the more general 'update' event, once Backbone is upgraded from Girder
+        this.listenTo(this.featuresets, 'remove', function () {
+            this.render();
+        }, this);
```

Comment:
You're right, but JS just ignores the extra arguments, so I never got a console error. Will fix.